index.html

```diff
@@ -0,0 +1,208 @@
+            </div>
+            <div class="flex_content">
+                <h2>Multi-Post Stories</h2>
+                <div class="canopy">
+                    <h3>FACEBOOK</h3>
+                    <ul class="canopy-list"><li>Full&nbsp;Stack&nbsp;Dev</li><li>2015</li></ul>
+                </div> 
+                <p class="card-info">
+                    Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                </p>
+                <ul class="list second">
+                <li>html</li>
+                <li class="ruby">Ruby on rails</li>
+                <li>css</li>
+                <li>javascript</li>
+                </ul>
+                <button class="see-more">See Project</button>
+            </div>
+        </div>
+          <!---The third card -->
+        <div class="flex">
+            <div class="flex_image">
+                <img class="mobile_image" src="images/card1-image" alt="Image for Card 1">
+                <img class="desktop_image" src="images/card3.png" alt="Image for desktop card 1">
+            </div>
+            <div class="flex_content">
+                <h2>Facebook 360</h2>
+                <div class="canopy">
+                    <h3>FACEBOOK</h3>
+                    <ul class="canopy-list"><li>Full&nbsp;Stack&nbsp;Dev</li><li>2015</li></ul>
+                </div> 
+                <p class="card-info">
+                    Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+                </p>
+                <ul class="list second">
+                <li>html</li>
+                <li class="ruby">Ruby on rails</li>
+                <li>css</li>
+                <li>javascript</li>
+                </ul>
+                <button class="see-more">See Project</button>
+            </div>
+        </div>
+           <!---The forth card -->
+        <div class="flex reverse">
+            <div class="flex_image">
+                <img class="mobile_image" src="images/card1-image" alt="Image for Card 1">
+                <img class="desktop_image" src="images/card4.png" alt="Image for desktop card 1">
+            </div>
+            <div class="flex_content">
+                <h2>Uber Navigation</h2>
+                <div class="canopy">
+                    <h3>Uber</h3>
+                    <ul class="canopy-list"><li>Full&nbsp;Stack&nbsp;Dev</li><li>2018</li></ul>
+                </div> 
+                <p class="card-info">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="list second">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+                </ul>
+                <button class="see-more">See Project</button>
```

Comment:
- [ ] It appears that the new changes that were introduced broke the design for your buttons. Right now your buttons have an overflow and the text does not fit within the button. Kindly fix this by reducing the size of the button text and the project will be ready for approval.

**This is what your buttons look like:**
![image](https://user-images.githubusercontent.com/86318284/178032333-f3735c81-8351-4070-8a6a-cdbb3cc6e0f2.png)

**This is how the buttons should look like:**
![image](https://user-images.githubusercontent.com/86318284/178032484-3e056ea4-2dbd-4aad-ba0e-b530b859ff52.png)

Reply:
Hello, I cannot identify the overflow in my buttons on my local machine, can you send the breakpoints please.